client/src/App.css

```diff
@@ -1,164 +1,181 @@
 main {
```

Comment:
@Pinaka-Pani-18 please make sure to set your editor to use 4 spaces for your tab character.